src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -151,4 +154,14 @@ namespace dpdk {
     return memsize;
   }
 
+  void eal::exit()
+  {
+    if (initialized == false || exit_thread)
+      return ;
+    dpdk::eal::execute_on_master([&exit_thread]() {
```

Comment:
i think there is no need to capture a global variable for changing it.

Reply:
sure